### DISCUSSION TITLE
Nushell enable extra features?

### DISCUSSION THREAD

**Discussion by jakevossen5** [Homebrew/discussions#1369] [Everyday usage]
CC @chenrui333

The current Nushell formula doesn't install with the `extra` features, which means that some commands don't work when installed with homebrew. [According to the maintainer](https://github.com/nushell/nushell/issues/3362#issuecomment-827861704), it would be best if it was distributed with these extra features enabled by default.

I suggest that we make building with `extra` features the default, and maybe add an option for only using base features?

**Comment by carlocab** [comment]
Please open a pull request to enable the extra features. You can omit the option for a base configuration, as formula options are not allowed in homebrew/core.

**Reply by jakevossen5** [reply]
Thanks, I opened https://github.com/Homebrew/homebrew-core/pull/76072
